spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala

```diff
@@ -30,7 +30,7 @@ import za.co.absa.enceladus.common.plugin.menas.MenasPlugin
 import za.co.absa.enceladus.common.{CommonJobExecution, Constants, RecordIdGeneration}
 import za.co.absa.enceladus.conformance.config.{ConformanceConfig, ConformanceConfigParser}
 import za.co.absa.enceladus.conformance.interpreter.rules.ValidationException
-import za.co.absa.enceladus.conformance.interpreter.{DynamicInterpreter, FeatureSwitches}
+import za.co.absa.enceladus.conformance.interpreter.{FeatureSwitches, DynamicInterpreter}
```

Comment:
Maybe unnecessary?

Reply:
Reverted as it was, thanks for noticing.